### PR TITLE
Feature/alcs 502

### PR DESCRIPTION
**Pull request by mhuseinov**
- conditions view pate
- transform single component link to condition into multi-select

### CODE REVIEW

File: services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision-v2.module.ts [hunk 1]
```diff
@@ -76,6 +77,7 @@ import { NaruSubtype } from '../../../portal/application-submission/naru-subtype
   controllers: [
     ApplicationDecisionV2Controller,
     ApplicationDecisionComponentController,
+    ApplicationDecisionConditionController,
```
Comment (sandratoh) [inline services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision-v2.module.ts:80]: Do we need to mock this in test or is the `.remove` basically it?

Comment (mhuseinov) [reply services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision-v2.module.ts:80]: Nope, this is a module and you do not need to mock a module in a test. Only injectables that are used in the service/controller need to be mocked. Also, you've highlighted the controller where .remove is for the service test 🙂

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-conditions/decision-conditions.component.spec.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ describe('DecisionConditionComponent', () => {
   beforeEach(async () => {
     mockDecisionService = createMock();
     mockDecisionService.$decision = new BehaviorSubject<ApplicationDecisionDto | undefined>(undefined);
-    mockDecisionService.$decisions = new BehaviorSubject<ApplicationDecisionDto[] | []>([]);
+    mockDecisionService.$decisions = new BehaviorSubject<ApplicationDecisionWithLinkedResolutionDto[]>([]);
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-conditions/decision-conditions.component.spec.ts:18]: It is used in: decison-conditions.component.ts -> ngOnInit ->  .pipe(combineLatestWith(this.decisionService.$decisions))

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-conditions/decision-condition/decision-condition.component.html [hunk 3]
```diff
@@ -13,17 +13,14 @@
...
-      [items]="selectableComponents"
-      formControlName="componentToCondition"
-      appendTo="body"
-      bindLabel="label"
-      bindValue="tempId"
-      required
-      placeholder="Component to Condition*"
-    ></ng-select>
+    <mat-form-field class="condition-grid-half-2" appearance="outline">
+      <mat-label>Component to Condition</mat-label>
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-conditions/decision-condition/decision-condition.component.html:17]: Maybe? The design shows multiple components linked and has this label: 'Component to Condition'

File: alcs-frontend/src/app/features/application/decision/conditions/condition/condition.component.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    <span [id]="condition.uuid + 'Description'" [ngClass]="{ 'ellipsis-3': !isReadMoreClicked }">{{
+      condition.description
+    }}</span>
+
+    <app-no-data *ngIf="!condition.description"></app-no-data>
+    <div class="read-more">
+      <a
+        (click)="onToggleReadMore()"
+        (keypress)="onToggleReadMore()"
+        [ngClass]="{ 'display-non': !isReadMoreVisible || !condition.description }"
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply alcs-frontend/src/app/features/application/decision/conditions/condition/condition.component.html:63]: good catch!

**Pull request merged**